neuro_san/internals/run_context/openai/openai_run_context.py

```diff
@@ -293,3 +293,22 @@ def get_journal(self) -> Journal:
         :return: The Journal associated with the instance
         """
         return self.journal
+
+    @staticmethod
+    def get_last_message_with_content(messages: List[Any]) -> object:
```

Comment:
This method moves here from message_utils.py.
This was the only place using it.